purchase_order_secondary_unit/reports/purchase_order_templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <!-- Header data -->
+        <th name="th_quantity" position="before">
+            <th name="th_secondary_unit" class="text-right">
```

Comment:
Since this PR https://github.com/odoo/odoo/pull/95450 Odoo use  BS5. According to the [Migration Guide](https://getbootstrap.com/docs/5.0/migration/#utilities), we must rename `.left-*` and `.right-*` to `.start-*` and `.end-*`.
```suggestion
            <th name="th_secondary_unit" class="text-end">
```